pkg/spi-shared/config/github.go

```diff
@@ -11,6 +11,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package oauth
+package config
 
-var GithubSaasHost = "github.com"
+var ServiceProviderTypeGitHub ServiceProviderType = "GitHub"
+
+var (
+	GithubSaasHost    = "github.com"
```

Comment:
I might be unnecessarily purist but I'd not make these public but rather "hide" them in the defaults or (if you accept my suggestion) service provider type instances.

E.g. instead of using this as `config.GithubSaasHost`, you'd `config.GithubServiceProviderDefaults.SaasHost` or `config.GithubServiceProviderType.SaasHost`.